pandas/core/series.py

```diff
@@ -2970,13 +2970,49 @@ def notnull(self):
 
     def dropna(self, axis=0, inplace=False, **kwargs):
         """
-        Return Series without null values
+        Return Series without null values.
+
+        In case of missing data, any or all values can be used reciprocally.
```

Comment:
Not sure what you mean by "reciprocally". I think this can section can be removed.